staker-rewards/rewards.js

```diff
@@ -269,6 +282,13 @@ function shortenSummaryValues(summary) {
   return summary
 }
 
+function writeOperatorsRewardsToFile(rewards) {
+  fs.writeFileSync(
+    "./generated-rewards/rewards-input.json",
```

Comment:
Maybe we could parametrize this via an environment variable with a default value?

Reply:
Good idea e11dd14